index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset=utf8>
+  NIN_WILL_REPLACE_THIS_TAG_WITH_AUTOGENERATED_COMMENT_TAGS
+  <title>My project</title>
```

Comment:
I'm surprised that didn't happen automatically seeing as the command I ran was `nin new re`